src/snowflake/snowpark/types.py

```diff
@@ -41,6 +42,23 @@ def __repr__(self):
     def is_primitive(self):
         return True
 
+    @classmethod
+    def type_name(cls) -> str:
+        return cls.__name__[:-4].lower()
+
+    def simple_string(self) -> str:
```

Comment:
why are these two methods not classmethod, and having different return types?

Reply:
Or you can do `simple_string = type_name` directly

Reply:
I am following pyspark's practice, i think they both have str as a return type?

Reply:
yeah, but why `def json_value(self) -> Union[str, Dict[str, Any]]:` is different?